drivers/sony/displays/bravia.cr

```diff
@@ -0,0 +1,229 @@
+require "placeos-driver/interface/powerable"
+require "placeos-driver/interface/muteable"
+require "placeos-driver/interface/switchable"
+
+class Sony::Displays::Bravia < PlaceOS::Driver
+  include Interface::Powerable
+  include Interface::Muteable
+
+  INDICATOR = "\x2A\x53" # *S
+  HASH      = "################"
+  ERROR     = 70
+
+  enum Inputs
+    Tv
+    Tv1
+    Tv2
+    Tv3
+    Hdmi
+    Hdmi1
+    Hdmi2
+    Hdmi3
+    Mirror
+    Mirror1
+    Mirror2
+    Mirror3
+    Vga
+    Vga1
+    Vga2
+    Vga3
+  end
+
+  INPUTS_CONVERT = {
+    "Tv"     => "0000",
+    "Hdmi"   => "1000",
+    "Mirror" => "5000",
+    "Vga"    => "6000",
+  }
+
+  CONVERT_LOOKUP = INPUTS_CONVERT.invert
+
+  include Interface::InputSelection(Inputs)
+
+  def switch_to(input : Inputs)
+    parsed_input = input.to_s.scan(/[^0-9]+|\d+/)
```

Comment:
Is this extracting the index? If so, this could possibly be done in a simpler non-regex based manner

Reply:
This would be good to address in a cleaner way, as well as isolated within `Input`